01-JS-DOM-APIs/javascript.js

```diff
@@ -0,0 +1,117 @@
+//1-4
+function show(){
+                  let obj = document.getElementById("hidden");
+                  if(obj.style.display == "block"){
+                        obj.style.display = "none";
+                        document.getElementById("buttonShow").value= "Show";
```

Comment:
since you are using the buttonShown node in both blocks, assign it to a variable as you did with your hidden section to make your code more efficient, searches in the DOM are really performance expensives.